src/main/java/mate/academy/spring/dto/request/UserRequestDto.java

```diff
@@ -1,7 +1,15 @@
 package mate.academy.spring.dto.request;
 
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import mate.academy.spring.validation.Email;
+
 public class UserRequestDto {
+    @Email
+    @Size(min = 6, max = 255)
```

Comment:
our regex is handling the size validation too so this size constraint is redundant